examples/counter_in/cont_acq_freq_wfm_large_range_2_cnt_impl_time.py

```diff
@@ -0,0 +1,39 @@
+"""Example of counter input frequency acquisition.
+
+This example demonstrates how to continuously measure buffered
+frequency using two counters on a Counter Input Channel. The
+Divisor, Maximum and Minimum Frequency Values, and the Edge
+Parameter are configurable.
+"""
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType, Edge, CounterFrequencyMethod, FrequencyUnits
+
+
+with nidaqmx.Task() as task:
+    chan = task.ci_channels.add_ci_freq_chan(
+        "Dev1/ctr0",
+        min_val=100000.0,
+        max_val=1000000.0,
+        units=FrequencyUnits.HZ,
+        edge=Edge.RISING,
+        meas_method=CounterFrequencyMethod.LARGE_RANGE_2_COUNTERS,
```

Comment:
lets remove this whole example. the non-default measurement methods are complex and mostly irrelevant these days. if people need them, they can figure it out.